src/steamship/agents/mixins/transports/telegram.py

```diff
@@ -12,6 +12,7 @@
 from steamship.agents.service.agent_service import AgentService
 from steamship.agents.utils import with_llm
 from steamship.invocable import Config, InvocableResponse, InvocationContext, post
+from steamship.utils.kv_store import KeyValueStore
 
 
 class TelegramTransportConfig(Config):
```

Comment:
Would we want to provide a default `''` string here for safety, simulate that it's now optional?